kibana/saved-objects/kibana-8-x-alerts.ndjson

```diff
@@ -0,0 +1,16 @@
+{"excludedObjects":[],"excludedObjectsCount":0,"exportedCount":15,"missingRefCount":0,"missingReferences":[]}
```

Comment:
shall we add a new line. look for other places as well

Reply:
Fixed with https://github.com/wso2-extensions/identity-elk-integration/pull/14